vocabularies/UI.xml

```diff
@@ -1401,6 +1401,37 @@ The trend is
         <Annotation Term="Core.IsLanguageDependent" />
       </Term>
 
+      <Term Name="InputMask" Type="UI.InputMaskType" Nullable="false" AppliesTo="Property Parameter">
+        <Annotation Term="Core.Description" String="Properties or parameters annotated with this term will get a mask in edit mode." />
+        <Annotation Term="Core.LongDescription">
+          <String>Input masks improve readability and help to enter data correctly. 
+So, masks can be especially useful for input fields that have a fixed pattern, e.g. DUNS numbers or similar. 
+[Here](../examples/UI.InputMask-sample.xml) you can find an example for this annotation</String>
+        </Annotation>
+      </Term>
+      <ComplexType Name="InputMaskType">
+        <Property Name="Mask" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="The mask to be applied to the property or the parameter" />
```

Comment:
```suggestion
          <Annotation Term="Core.Description" String="The mask to be applied to the property or the parameter" />
          <Annotation Term="Core.LongDescription">
            <String>Symbols in the mask for which there is an `InputMaskRule` are initially displayed using the `Placeholder`, and what the user types in their position must match the associated regular expression. Other symbols are read-only and the cursor automatically moves over them as the user types.</String>
          </Annotation>
```

Reply:
This sounds even for me, who created the term, far too complex - maybe we can write something similar in the example as an explanation. Maybe we can ask a UA colleague to make a proposal.

Reply:
We must specify the expected UI behavior somewhere.

Reply:
yes - definitely.